contract_wrapper/KlerosWrapper.js

```diff
@@ -47,15 +47,39 @@ class KlerosWrapper extends ContractWrapper {
 
   /**
    * Get disputes
+   * @param account address of user
    * @return objects[]
    */
-  getDisputes = async (
-    account = this._web3Wrapper.getAccount(0)
+  getDisputesForUser = async (
+    account = this._Web3Wrapper.getAccount(0)
   ) => {
     const profile = await this._StoreProvider.getUserProfile(account)
     if (!profile) throw new Error("No user profile for " + account)
     return profile.disputes
   }
+
+  /**
+   * Get dispute by id FIXME isn't calling blockchain
+   * @param disputeHash hash of the dispute
+   * @param account address of user
+   * @return objects[]
+   */
+  getDisputeByHash = async (
```

Comment:
I think we need a method `getDisputeById` to get the dispute from the smart contract.